packages/neos-ui/package.json

```diff
@@ -11,11 +11,11 @@
     "typescript": "^4.6.4"
   },
   "dependencies": {
-    "@ckeditor/ckeditor5-highlight": "16.0.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.35",
-    "@fortawesome/free-brands-svg-icons": "^5.15.3",
-    "@fortawesome/free-regular-svg-icons": "^5.15.3",
-    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@ckeditor/ckeditor5-highlight": "42.0.2",
+    "@fortawesome/fontawesome-svg-core": "^6.5.2",
```

Comment:
Is it intended that you upgrade CKEditor and font awesome at once?

Reply:
Removed the fontawesome update :).